gaj23/rancid
# Gaj/styling/header touchup

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*

This PR completes [Iteration 1](https://frontend.turing.io/projects/module-3/rancid-tomatillos-v3.html) as outlined by Turing School of Software and Design. We've take a mobile first approach, using mock data to initially display our movies. 

## *Instructions for reviewer:*
*Where should the reviewer start?*

Please look at our components files to see set up. 

*How should this be manually tested?*

Please run locally (clone down and `npm start` in terminal) to view in browser

*Any background context you want to provide?*

Please see [iteration 1](https://frontend.turing.io/projects/module-3/rancid-tomatillos-v3.html)

*What are the relevant tickets?*

Issues #1 & #3

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

There are small styling concerns at this point. 

Things to keep in mind for the future, however, have to do with displaying the film, or the details of a movie, and then navigating back to the main page. 

Additionally, testing. How should we do testing?

## Checklist:
